### PR TITLE
9 delete by comment

### PR DESCRIPTION
**Pull request by Gazdean**

### CODE REVIEW

File: db/models/articles-model.js [hunk 1]
```diff
@@ -97,6 +97,69 @@ exports.createArticleComment = (article_id, username, body) => {
...
+        SET
+        votes = $1
+        WHERE article_id = $2
+        RETURNING *;
+        `, [updatedVotes, article_id]
+       )      
+    }) 
+}
+exports.removeCommentById = (comment_id) => {
+   return db
```
Comment (RichardNeat) [inline db/models/articles-model.js]: No need for a separate database query here to check for the article existing. We can assess the return value from the query to determine if the query has been successful. Take a look at the response that comes back from the delete query, console log it and see what the difference is when it's gone well compared to when it hasn't worked.

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Minor change requested please. Drop me a screenshot once that's done instead of resubmitting or let me know if you're struggling :)

File: db/models/articles-model.js (hunk 1 above)
Comment (Gazdean) [reply db/models/articles-model.js]: done

**Comment by Gazdean** [comment]
merged
deleted branch

**Pull request merged**